src/datamigration/azext_datamigration/manual/_params.py

```diff
@@ -52,6 +52,15 @@ def load_arguments(self, _):
         c.argument('database_deny_list', nargs='+', help='Space separated list of names of databases to not be considered for SKU recommendation. Only set one of the following or neither: databaseAllowList, databaseDenyList. Default: null.')
         c.argument('config_file_path', type=str, help='Path of the ConfigFile')
 
+    with self.argument_context('datamigration login-migration') as c:
+        c.argument('src_sql_connection_str', nargs='+', help='Connection string(s) for the source SQL instance(s), using the formal connection string format.')
+        c.argument('tgt_sql_connection_str', type=str, help='Connection string(s) for the target SQL instance(s), using the formal connection string format.')
+        c.argument('csv_file_path', type=file_type, completer=FilesCompleter(), help='Location of CSV file of logins. Use only one parameter between this and listOfLogin.')
```

Comment:
You need to import `file_type` from cli core. Not simply modify what I suggested! I'm just giving suggestions not developing for you!

Reply:
> You need to import `file_type` from cli core. Not simply modify what I suggested! I'm just giving suggestions not developing for you!

thanks for the tips. I added it